gcpdiag/queries/gke.py

```diff
@@ -268,6 +268,9 @@ def has_logging_enabled(self) -> bool:
   def has_monitoring_enabled(self) -> bool:
     return self._resource_data['monitoringService'] != 'none'
 
+  def has_authenticator_group_enabled(self) -> bool:
```

Comment:
Could you add a test for this new method in gke_test.py?

Reply:
I've written the test but it'll require additional data in the json dumps so I'll reach out to dwes@google.com as instructed in the README. Will update accordingly.